### PR TITLE
fix command description typo on src/cogs/esports/slash/scrims.py

### LINKED ISSUE #135: fix command description typo
The user text Should be `The user you want to unban from scrims.`
And,
The reason text should be `The reason for unbanning the user.`

_Originally posted by @aayyuush99 in https://github.com/quotientbot/Quotient-Bot/pull/134#discussion_r1228411353_

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
Fixes #135 

### Description:

Hi [deadaf](https://github.com/deadaf), I have tried to fix the type mentioned before, please check it out

___

### Checklist:
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Review by deadaf — CHANGES_REQUESTED** [review]
Hello @viktoriussuwandi , Thanks for the contribution. You did the reason description right but you forgot the user, it still stays The user you want to ban from scrims, it should actually be 
"The user you want to unban from scrims".

File: src/cogs/esports/slash/scrims.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ async def can_use_command(self, interaction: discord.Interaction) -> bool:
         return True
 
     @app_commands.command()
-    @app_commands.describe(user="The user you want to ban from scrims.", reason="The reason for banning the user.")
+    @app_commands.describe(user="The user you want to ban from scrims.", reason="The reason for unbanning the user.")
```
Comment (deadaf) [inline src/cogs/esports/slash/scrims.py:39]: ~~The user you want to ban from scrims.~~

The user you want to ban from scrims.  ✅

Comment (viktoriussuwandi) [reply src/cogs/esports/slash/scrims.py]: sorry miss that part, it looks god now, right [deadaf](https://github.com/deadaf) ?

**Review by deadaf — APPROVED** [review]
thanks @viktoriussuwandi !

**Pull request merged**